ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/ip-nat.cfg

```diff
@@ -22,6 +22,14 @@ ip nat profile NAT-PROFILE-NO-VRF-2
    ip nat source static 3.0.0.6 22 4.0.0.6 23 protocol tcp group 2 comment Comment Test
    ip nat source static 3.0.0.7 access-list ACL21 4.0.0.7
    ip nat source ingress static 3.0.0.8 4.0.0.8
+   ip nat destination static 1.0.0.1 2.0.0.1
```

Comment:
This does not exactly match with EOS but tried to simplify.

Reply:
yes, EOS sorts the config on the basis of `ip address` So the `source` and `destination` order is mixed -
```
interface Ethernet2
   ip nat destination static 1.0.0.1 2.0.0.1
   ip nat source static 3.0.0.2 22 4.0.0.2
   ip nat destination static 3.0.0.5 1.1.1.1
   ip nat source static 3.0.0.7 access-list ACL21 4.0.0.7
   ip nat source static 3.0.0.6 22 4.0.0.6 23 protocol tcp group 2 comment Comment Test
```
and we are sorting both `source` and `destination` based on `original_ip`, so `source` comes first then `destination`

Reply:
Yes.

Reply:
I have tried to update the template to sort similar to EOS. Please take a look again. Thanks.

Reply:
Sequence for below has mismatch. I am trying this.
EOS:
   ip nat source static 3.0.0.5 22 4.0.0.5 23 protocol tcp group 1
   ip nat destination static 1.0.0.5 22 2.0.0.5 23 protocol tcp group 1
   ip nat source static 3.0.0.6 22 4.0.0.6 23 protocol tcp group 2 comment Comment Test
   ip nat destination static 1.0.0.6 22 2.0.0.6 23 protocol tcp group 2 comment Comment Test
   AVD:
   ip nat source static 3.0.0.5 22 4.0.0.5 23 protocol tcp group 1
   ip nat source static 3.0.0.6 22 4.0.0.6 23 protocol tcp group 2 comment Comment Test
   ip nat destination static 1.0.0.5 22 2.0.0.5 23 protocol tcp group 1
   ip nat destination static 1.0.0.6 22 2.0.0.6 23 protocol tcp group 2 comment Comment Test

Reply:
Fixed this part.

Reply:
Thanks. @Shivani-gslab can you take a look again, since we have fixed the order now.

Reply:
Looks good!